templates/emails/reviewers_group_invitation.rst

```diff
@@ -0,0 +1,18 @@
+Hello,
+
+We hope you are in good health and doing well.
```

Comment:
Not sure why the linebreak here might look odd on email.

Reply:
Yeah will remove that.